tests/test_tlsfuzzer_expect.py

```diff
@@ -1285,52 +1285,140 @@ def test_process_with_tls13_unallowed_extension(self):
         with self.assertRaises(AssertionError):
             exp.process(state, server_hello)
 
-    def test_process_with_tls_1_3_no_downgrade_protection(self):
-        exp = ExpectServerHello(version=(3, 4), server_max_protocol=(3, 4))
+    def test_process_with_tls_1_3_in_legacy_version(self):
+        exp = ExpectServerHello()
 
         state = ConnectionState()
         client_hello = ClientHello()
         ciph = CipherSuite.TLS_EMPTY_RENEGOTIATION_INFO_SCSV
         client_hello.cipher_suites = [4, ciph]
         state.handshake_messages.append(client_hello)
         state.msg_sock = mock.MagicMock()
+        ext = []
+        ext.append(SrvSupportedVersionsExtension().create((3, 4)))
 
         msg = ServerHello().create(version=(3, 4),
```

Comment:
is version here ok ?

Reply:
yes, this is exactly to verify that the server doesn't advertise the negotiated version incorrectly